### DISCUSSION TITLE
Bean's Discord?

### DISCUSSION THREAD

**Discussion by layncemc** [BeansGalaxy/Beans-Backpacks-2#210] [Feedback] [answered]
I would love to see more announcements of projects, or perhaps a suggestion page on discord since I feel more people regular that app than github :)

**Comment by BeansGalaxy** [comment, marked as answer]
Truthfully I don't really know how to use discord lmao, I don't have any online friends and if I do they're probably bots. If the community grows first without it then I'll definitely set up a page but for now I'm not active enough to justify it.

**Comment by TheoClouds** [comment]
I’d love this! I was looking for one on the mod page but saw it’s just the GitHub and wiki.
